src/app/dynamic-forms/fields/cinchy-my-format.ts

```diff
@@ -0,0 +1,11 @@
+export const MY_FORMATS = {
```

Comment:
Name this to something more appropriate like

display-format.ts

and `export const DisplayFormats`

Reply:
Resolved